services/power.md

```diff
@@ -4,6 +4,35 @@
 
 A power-provider service.
 
+## Power negotiation protocol
+
+The purpose of the power negotiation is to ensure that there is no more than ~500mA
+delivered to the power rail.
+This is realized by limiting the number of enabled power provider services to 1.
+
+Note, that it's also possible to have low-current power providers, which do not
+run a power provider service.
+These are **not** accounted for in the power negotiation protocol.
+
+* Upon reset, a power service enables itself.
```

Comment:
does "enable" mean power the rail?

Reply:
does it enter in protection mode here too?

Reply:
yeah, thinking about this...